Frameworks/AudioToolbox/CAFDecoder.mm

```diff
@@ -184,68 +184,45 @@ uint64_t int64Swap(uint64_t val) {
     return ret;
 }
 
-enum {
-    kAudioCodecProduceOutputPacketSuccess,
-    kAudioCodecProduceOutputPacketNeedsMoreInputData,
-    kAudioCodecProduceOutputPacketSuccessHasMore
-} Result;
-
-int CAFDecoder::GetUsedInputBufferByteSize(EbrFile* fpIn) {
-    int cur = EbrFtell(fpIn);
-    EbrFseek(fpIn, 0, SEEK_END);
-    int left = EbrFtell(fpIn) - cur;
-    EbrFseek(fpIn, cur, SEEK_SET);
-
-    return left;
-}
-
-Byte* CAFDecoder::GetBytes(EbrFile* fpIn, int len) {
+Byte* CAFDecoder::GetBytes(NSInputStream* stream, int len) {
```

Comment:
Changes to this file don't have much collateral, please be sure to validate internal apps.